cloudomate/util/recaptchasolver.py

```diff
@@ -0,0 +1,142 @@
+import json, requests
+from time import sleep
+
+
+"""
+use this already generated API key for testing: fd58e13e22604e820052b44611d61d6c
+
+Usage:
+
+Feed Anti-captcha account API key to solver 
+rc_solver = reCaptchaSolver("fd58e13e22604e820052b44611d61d6c")
+
+Find out how much money is left
+temp = rc_solver.getBalance()
+
+Provide solver method with Google ReCapthca URL and ReCaptcha website key usually found as data-sitekey attribute in an HTML element in the website containing the ReCaptcha
+wUrl = "http://http.myjino.ru/recaptcha/test-get.php"
+wKey = "6Lc_aCMTAAAAABx7u2W0WPXnVbI_v6ZdbM6rYf16"
+rc_solution = rc_solver.solveGoogleReCaptcha(wUrl,wKey)
+print(rc_solution)
+
+"""
+
+class reCaptchaSolver:
+
+    __clientKey = "not set"
+
+    def __init__(self, cKey):
+        self.__clientKey = cKey
+
+    def getBalance(self):
```

Comment:
It seems that there's a lot of overlap between these two files. Please make the `reCaptchaSolver` a subclass of `CaptchaSolver`. This would clean up the code tremendously.